server/gql/property.graphql

```diff
@@ -65,6 +65,7 @@ enum PropertySchemaFieldUI {
   MARGIN
   PADDING
   ZOOMLEVEL
+  DYNAMIC_PROPERTY_SELECTOR
```

Comment:
Still feel a little bit too long, could you use `PROPERTY_SELECTOR` instead? Please update on other places as well.